src/main/java/domain/Car.java

```diff
@@ -0,0 +1,32 @@
+package domain;
+
+public class Car {
+
+    public static final int CAR_MOVE_CHECK = 4;
+    public static final int CAR_ADVANCE = 1;
+    public static final int CAR_STOP = 0;
+
+    private CarDistance carDistance;
+
+    public Car() {
+        carDistance = new CarDistance();
+    }
+
+    public void carMoveBehavior(int randomNumber) {
+
+        if (randomNumber >= CAR_MOVE_CHECK) {
+            move(()->CAR_ADVANCE);
+            return;
+        }
+
+        move(()->CAR_STOP);
```

Comment:
```suggestion
            move();
        }
```

와 어떠한 차이가 있을까요?
지금과 같은 구조로 설계하신 이유가 있을까요?

Reply:
public void move(CarBehavior carBehavior) {
        carDistance.addCarDistance(carBehavior.carMove());
}

와 같이 move 함수가 CarBehavior 라는 interface를 매개변수로 받고 있습니다.
4이상일 경우에는 1을 add 하고 그 외에는 0을 add 하는 구조로 설계를 하였습니다.